### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -69,26 +69,25 @@ The meeting recording option is a setting at the Teams policy level. The followi
 > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:69]:
```suggestion
> [!NOTE]
```

File: Teams/tmr-meeting-recording-change.md [hunk 2]
```diff
@@ -69,26 +69,25 @@ The meeting recording option is a setting at the Teams policy level. The followi
...
 > Teams meeting policy changes take awhile to propagate. Check back after a few hours of setting it, then sign out and sign in again.
 
-1. Install Skype For Business Online PowerShell.
+1. Install Teams PowerShell PowerShell.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:72]:
```suggestion
1. Install Teams PowerShell module.
```

File: Teams/tmr-meeting-recording-change.md [hunk 3]
```diff
@@ -69,26 +69,25 @@ The meeting recording option is a setting at the Teams policy level. The followi
...
    > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector. See [Manage Skype for Business Online with PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/manage-skype-for-business-online-with-microsoft-365-powershell?view=o365-worldwide&preserve-view=true).
 
-    1. Download [Skype for Business Online PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/manage-skype-for-business-online-with-microsoft-365-powershell?view=o365-worldwide&preserve-view=true).
+1. Launch PowerShell as an admin.
 
-    1. Follow the prompts to install it.
+2. Install [Teams PowerShell module](https://docs.microsoft.com/microsoftteams/teams-powershell-install).
 
-    1. Restart your machine.
+3. Import the MicrosoftTeams module and sign in as a Teams admin.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:81]:
```suggestion
3. Import the Microsoft Teams module and sign in as a Teams admin.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**